coralproject/cay
# Form builder button reorder

## What does this PR do?
- Fixes: https://github.com/coralproject/ask/issues/60
## How do I test this PR?
- Go to a form
- Click on Edit Form
- Publish, preview and Save should be on the sidebar now.
- Clicking publish should show you a Modal with all the publishing options

@coralproject/frontend
